mail-client/geary/geary-3.34.1.ebuild

```diff
@@ -0,0 +1,93 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+VALA_MIN_API_VERSION="0.40"
+
+inherit gnome.org gnome2-utils meson vala xdg
+
+DESCRIPTION="A lightweight, easy-to-use, feature-rich email client"
+HOMEPAGE="https://wiki.gnome.org/Apps/Geary"
+
+LICENSE="LGPL-2.1+ BSD-2 CC-BY-3.0 CC-BY-SA-3.0" # code is LGPL-2.1+, BSD-2 for bundled snowball-stemmer, CC licenses for some icons
+SLOT="0"
+
+IUSE="unwind"
+
+KEYWORDS="~amd64 ~x86"
+
+# for now both enchants work
+# FIXME: add valadoc support
+
+DEPEND="
+	>=dev-libs/glib-2.54:2
+	>=gui-libs/libhandy-0.0.9:0.0=
+	>=x11-libs/gtk+-3.22.26:3[introspection]
+	>=net-libs/webkit-gtk-2.20:4=[introspection]
+	app-text/gspell[vala]
+	app-text/iso-codes
+	>=dev-db/sqlite-3.12:3
+
+	>=net-libs/libsoup-2.48:2.4[introspection]
+	net-libs/gnome-online-accounts
+	>=net-mail/ytnef-1.9.3
```

Comment:
net-mail/ytnef can be made optional using `-Dtnef-support` in emesonargs (tested with geary-3.34.0).

Reply:
do you have somewhere ebuild with this change?